app/src/main/res/drawable/ic_whats_new.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This icon is called `subscriptions` [here](https://fonts.google.com/icons?selected=Material+Icons&icon.query=subscriptions), so please keep the original name for consistency